src/test/java/com/iota/iri/network/impl/TransactionRequesterWorkerImplTest.java

```diff
@@ -0,0 +1,142 @@
+package com.iota.iri.network.impl;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+import static org.mockito.Mockito.*;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.Random;
+
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import com.iota.iri.controllers.TipsViewModel;
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.controllers.TransactionViewModelTest;
+import com.iota.iri.model.Hash;
+import com.iota.iri.model.persistables.Transaction;
+import com.iota.iri.network.Node;
+import com.iota.iri.network.TransactionRequester;
+import com.iota.iri.network.impl.TransactionRequesterWorkerImpl;
+import com.iota.iri.storage.Tangle;
+import com.iota.iri.utils.Converter;
+
+public class TransactionRequesterWorkerImplTest {
+    
+    //Good
+    private static final TransactionViewModel TVMRandomNull = new TransactionViewModel(
+            getRandomTransaction(new Random()), Hash.NULL_HASH);
+    private static final TransactionViewModel TVMRandomNotNull = new TransactionViewModel(
+            getRandomTransaction(new Random()), TransactionViewModelTest.getRandomTransactionHash());
+    private static final TransactionViewModel TVMAll9Null = new TransactionViewModel(
+            get9Transaction(), Hash.NULL_HASH);
+    
+    //Bad
+    private static final TransactionViewModel TVMAll9NotNull = new TransactionViewModel(
+            get9Transaction(), TransactionViewModelTest.getRandomTransactionHash()); 
+    
+    private static Tangle tangle;
+    private static Node node;
+    private static TipsViewModel tvm;
+    
+    private static TransactionRequester requester;
+    private static TransactionRequesterWorkerImpl worker;
+    
+    @Rule 
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+    
+    @BeforeClass
+    public static void setUp(){
+        requester = mock(TransactionRequester.class);
+        requester.init(0); //Dont drop anything
```

Comment:
This line isn't doing anything because you are using a mock.
Perhaps you wanted to use a `spy` here?
Or use the actual object with mocked dependencies?